docs/index.md

```diff
@@ -91,6 +106,8 @@ There are 2 types of Optimization Functions for TUI's Transfer Service:
 | Minimize Distance     | Distance = total distance travelled                          | Destinations where most transfers use TUI's own fleet        | Mallorca            |
 | Minimize Cost         | Cost = a function of vehicle type, number of passengers driven, and areas driven (designed to mirror actual pricing structure with suppliers) | Destinations where most transfers use vehicles contracted from suppliers | Zakynthos           |
 
+Jamie: Didn't we introduce another option?  Do we want to provide all of these options to Cyberlogic?
+
```

Comment:
I don't think we have yet, but I think we will try to emphasize some of the flexibility here in the final doc